src/components/Landing/Landing.jsx

```diff
@@ -1,20 +1,30 @@
 import React from "react";
+import RENDER from "SHARED/renderUtils.jsx";
+import { s } from "./";
 
 export default function Landing(){
+
+	const bodyData = `When a king falls, another will rise. But who will it be?\nGet a closer look at `;
```

Comment:
The 
```suggestion
	const bodyData = `When a king falls, another will rise. But who will it be?
	Get a closer look at `;
```
If we use a template literal then we don't have to use `\n` 🙌

Reply:
Also, rather than split out the link like this, let's include it as part of the text - remember that we want to move all of the content like this to a `data.js` that we import, and that should all be treated as if it's client-facing via CMS or what-have-you.  We could do something like:

```
const data = {
    heading: Vikings,
    subheading: Desecend into Darkness,
    body: `When a king falls, another will rise. But who will it be?
Get a closer look at [Vikings Season 5](#about)`
};

export default data;
```

By using markdown we can put links in our data and then parse them out later - you can make your own `parseMarkdown` util if you'd like to - if not, a good tool for this is [marked](https://www.npmjs.com/package/marked) - which we can install and then import here.

Reply:
I did it that ^ way because I couldn't figure out a good way of parsing it and pulling the anchor out (i spent quite a bit of time on it!). 

I gave `marked` a try but after about 10 mins of head-scratching I decided to do it myself.

New findLink method in the render utils. I'm sure there's a nicer way of doing it but it was good to work it out and using a few things i've picked up :) (selective rendering woo!)

Reply:
@pookage